src/commons/BrowserUtils.ts

```diff
@@ -414,14 +414,30 @@ export namespace BrowserUtils {
       'Failed to find elements'
     );
   }
+    /**
+     * Hover over an element by given selector
+     *
+     * Note: Uses moveToObject method that is currently deprecated
+     * @param selector selector of an element to hover
+     */
+    export function hover(selector: string): void {
+        Reporter.debug(`Move to an element '${selector}'`);
+        isDisplayed(selector);
+        tryBlock(
+          () => $(selector).moveTo(),
+          `Failed to hover over '${selector}' at location '${JSON.stringify(
+            location
+          )}'`
+        );
+    }
 
   /**
-   * Hover over an element by given selector
+   * Hover by element location - hover an element by given selector by extracting element location
    *
    * Note: Uses moveToObject method that is currently deprecated
    * @param selector selector of an element to hover
    */
-  export function hover(selector: string): void {
+  export function hoverOverElementLocation(selector: string): void {
```

Comment:
How does it differ from hover function ? why do we need both ?

Reply:
We "need" both because WDIO enables us to do both $(selector).moveTo() OR element.moveTo(location.x, location.y) in some cases under the console the first work better and in some cases the second worked better. if you think we should loose one of then tell me and ill remove.

Reply:
in what case hover does not work? How user should to know which one to use?

Reply:
Removed by location.

Reply:
is there tests that will fail now?

Reply:
no